lib/ex_okex/ws.ex

```diff
@@ -82,11 +83,15 @@ defmodule ExOkex.Ws do
         |> handle_response(state |> inc_heartbeat())
       end
 
-      def handle_frame({:binary, compressed_data}, state) do
-        compressed_data
-        |> :zlib.unzip()
-        |> Jason.decode!()
-        |> handle_response(state)
+      def handle_frame({:binary, compressed_data}, %{channels: channels} = state) do
+        case compressed_data |> :zlib.unzip() |> Jason.decode!() do
+          %{"event" => "login", "success" => true} ->
```

Comment:
I think we should only do this for `private channel` only right. For public one, probably no need

Reply:
As for now, we don't distinguish private and public channels. I propose to leave it as it is and improve it in the next task